### ISSUE TITLE
[FEATURE] Update the "online grayskull " tool website

### ISSUE THREAD

**Issue by ForgottenProgramme** [conda/grayskull#428]
**End Result**
- the grayskull website can be used to generate CRAN recipes as well
- Presently the website asks for `project name` and `version`. Now that we support CRAN also, the website should first ask `repository` (CRAN or PyPI), then `project name` and `version`

**Comment by marcelotrevisani** [comment]
yeah, that needs some work, but it will not be my priority for now. I would rather focus on adding support for poetry and flint first, after that I can rework the webapi.

**Comment by bollwyvl** [comment]
A semi-crazy direction to go would be getting the whole `grayskull` shooting match running in the browser under WASM, 
and hosting on a simple static ReadTheDocs page or GitHub pages. This would shift much of the burden from a backend process to CDN-mirrored static assets.

Either `pyodide` or `xeus-python` and [emscripten-forge](https://github.com/emscripten-forge/recipes/issues) could likely do the job.

The only missing dep from `pyodide`, at present is `rapidfuzz`, which appears to also be missing from `emscripten-forge`. Getting it into one of those channels probably wouldn't be a huge lift, but then maybe having `rapidfuzz` be hidden behind a conditional dependency, and having a fallback pure-python implementation would get it sorted faster.

At present, `pyodide`'s solver already relies on the PyPI JSON API, so no worries there, but as the CRAN integration is scraping-based, it would likely cause more issues when actually running.
